zero/correct_mj.c

```diff
@@ -79,17 +79,17 @@ gkyl_correct_mj_new(const struct gkyl_rect_grid *grid,
 }
 
 void 
-gkyl_correct_mj_fix_m0(gkyl_correct_mj *cmj, 
+gkyl_correct_mj_fix_n_stationary(gkyl_correct_mj *cmj, 
   struct gkyl_array *fout,
-  const struct gkyl_array *m0, const struct gkyl_array *m1i,
+  const struct gkyl_array *n, const struct gkyl_array *vbi,
```

Comment:
We have recently tried to adopt a standard where we don't use single letter variable names, so if possible, I believe this n -> n_stationary.